app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @public_recipes = Recipe.where('public = true').order(id: :desc).includes(:foods).includes(:user)
+  end
```

Comment:
- [ ] Currently, I'm getting an error when I'm trying to click on `public recipe` which is created by another user. Please fix it so that anyone can visit the public recipe detail.

![Screenshot from 2022-04-15 17-03-55](https://user-images.githubusercontent.com/22434660/163567161-f39a5cd0-28bf-4443-9729-7e4406776c6b.png)